### PR TITLE
feat(site): embed users page in management settings

### PR DESCRIPTION
**Pull request by aslilac**
Part of #13957

Screenshot 2024-07-24 at 2 47 18 PM

### CODE REVIEW

**Review by code-asher — APPROVED** [review]

**Comment by code-asher** [comment]
I completely missed the missing button :facepalm: 

I see the create user page navigates to `/user`, should we make it `/deployment/user` for multi-org?  I know the redirect catches it anyway so probably nbd, I did notice it causes two user API requests though (but the first gets canceled).

**Comment by aslilac** [comment]
> I completely missed the missing button 🤦
> 
> I see the create user page navigates to `/user`, should we make it `/deployment/user` for multi-org? I know the redirect catches it anyway so probably nbd, I did notice it causes two users API requests though (but the first gets canceled).

Yeah, and I should also have it send you to /deployment/users/create instead of just /users/create. 😄

File: site/src/pages/CreateUserPage/CreateUserPage.tsx [hunk 1]
```diff
@@ -32,10 +32,10 @@ export const CreateUserPage: FC = () => {
         onSubmit={async (user) => {
           await createUserMutation.mutateAsync(user);
           displaySuccess("Successfully created user.");
-          navigate("/users");
+          navigate("..", { relative: "path" });
```
Comment (code-asher) [inline site/src/pages/CreateUserPage/CreateUserPage.tsx:35]: Ooo such a good idea, relative is smart

**Review by BrunoQuaresma — APPROVED** [review]
Everything looks good to me.

**Pull request merged**